exercises/practice/resistor-color/resistor-color_test.sql

```diff
@@ -0,0 +1,47 @@
+-- Create database:
+.read ./create_fixture.sql
+
+-- Read user student solution and save any output as markdown in user_output.md:
+.mode markdown
+.output user_output.md
+.read ./resistor-color.sql
+.output
+
+-- Create a clean testing environment:
+.read ./create_test_table.sql
+
+-- Comparison of user input and the tests updates the status for each test:
+UPDATE "test_color_code"
+SET status = 'pass'
+FROM (SELECT color, result FROM color_code) AS l
+WHERE (l.color, l.result) = (test_color_code.color, test_color_code.result);
+-- Update message for failed tests to give helpful information:
+UPDATE test_color_code
+SET message = 'Result for ' || l.color || ' is "' || l.result || '", but should be: "' || test_color_code.result || '"'
+FROM (SELECT color, result FROM color_code) AS l
+WHERE l.color = test_color_code.color AND test_color_code.status = 'fail';
+
+-- Comparison of user input and the tests updates the status for each test:
+UPDATE test_colors
+SET status = 'pass'
+FROM (SELECT GROUP_CONCAT("color" ORDER BY "index") as "result" FROM "colors") AS l
```

Comment:
No "order by" in sqlite group_concat: https://sqlite.org/lang_aggfunc.html#group_concat

Reply:
```
sqlite> SELECT GROUP_CONCAT("color" ORDER BY "index") AS "result" FROM "colors";
+------------------------------------------------------------+
|                           result                           |
+------------------------------------------------------------+
| black,brown,red,orange,yellow,green,blue,violet,grey,white |
+------------------------------------------------------------+

sqlite> SELECT GROUP_CONCAT("color") AS "result" FROM "colors";
+------------------------------------------------------------+
|                           result                           |
+------------------------------------------------------------+
| blue,orange,violet,red,brown,black,white,green,yellow,grey |
+------------------------------------------------------------+

Reply:
I'm not sure what version I'm running that does this but the CI doesn't like it. I'll see if I can find another way to make this test work.